index.js

```diff
@@ -25,12 +25,13 @@ function translate(text, opts) {
 
     opts.from = opts.from || 'auto';
     opts.to = opts.to || 'en';
+    opts.region = opts.region || 'com';
```

Comment:
The opts.region option has been added to the function parameters and a default value of 'com' is set.